src/content/developers/docs/data-and-analytics/block-explorers/index.md

```diff
@@ -25,46 +24,33 @@ You should understand the basic concepts of Ethereum so you can make sense of th
 
 ## Data {#data}
 
-Ethereum is transparent by design so everything is verifiable. Block explorers provide an interface for getting this information. And this is for both the main Ethereum network and the testnets, should you need that data.
+Ethereum is transparent by design so everything is verifiable. Block explorers provide an interface for getting this information. And this is for both the main Ethereum network and the testnets, should you need that data. Data is divided into execution data and consensus data. The execution data refers to the transactions that have been executed in a specific block. The consensus data refers to the blocks themselves and the validators who proposed them.
```

Comment:
> Data is divided into execution data and consensus data. The execution data refers to the transactions that have been executed in a specific block. The consensus data refers to the blocks themselves and the validators who proposed them.

Isn't this something we're assuming people understand as a prerequisite?

Reply:
just reiterating to justify the layout of the rest of the page.